src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -57,6 +58,45 @@ public void init() {
         DenizenCore.commandRegistry.registerCommand(MythicSkillCommand.class);
         new MythicMobsLoaders().RegisterEvents();
 
+        EntityTag.tagProcessor.custommatchers.add((entityTag, matcher) -> {
+            if (!matcher.equals("mythic_mob")) {
+                return null;
+            }
+            return entityTag.getUUID() != null && getMobManager().isActiveMob(entityTag.getUUID());
+        });
+        EntityTag.tagProcessor.custommatchers.add((entityTag, matcher) -> {
+            if (!matcher.startsWith("mythic_mob:")) {
+                return null;
```

Comment:
This should ideally be one registered function per type, with positive `if (equals) { return validResponse; }`, rather than split into two with negative checks

Reply:
Should be done